packages/website/src/theme/MDXComponents/ConfigurationContents.tsx

```diff
@@ -0,0 +1,22 @@
+import CodeBlock from '@theme/CodeBlock';
+import { configs } from '@typescript-eslint/website-eslint';
```

Comment:
i wonder if we could just generate this like we generate the configs?
or is that what you're trying to avoid?

Reply:
I was trying to avoid that, yeah. I was hoping we could have just the compiled outputs so the website wouldn't have to load as much data.

But I don't have the time+energy to debug through the weird Rollup issue. I'll just link to the files for now.